scenarios/27_Orannon.cfg

```diff
@@ -407,110 +307,28 @@
         team_name=bad
         canrecruit=yes
         side=8
-        {GOLD 310 330 350}
+        {GOLD 300 325 350}
         {INCOME 8 10 12}
```

Comment:
See the thing that happens with Grekulak is he arrives on the battlefield, sends out his army, and then runs out of gas once you kill everyone and dies lamely. What I suggest is setting his income when he arrives, and you can lower his starting gold if you like, e.g. to {200 250 300}. Then, make his income much higher, like {16 20 24}. The same goes for sides 7 and 9. This way you actually have to push to kill him.

Reply:
I mean I had a hard enough time just surviving his initial sending out of his army, so I don't really want to buff him TOO much... also because of the `RISE_UP_RISE_UP` macro he can still build his army without actually recruiting (something that happened to me a lot because I let my units die a lot), so I don't really want to raise his income THAT much... maybe just `{10 12 14}`?

Reply:
You can make it vary way more on difficulty then. 12 16 20 or 10 15 20.

Reply:
ok I went with `{10 15 20}` in 4bdbac0